### ISSUE TITLE
Improve checking of alt text 

### ISSUE THREAD

**Issue by maelle** [ropensci-org/roblog#25]
https://matt-dray.github.io/altcheckr/

**Comment by maelle** [comment]
@steffilazerte @yabellini when you review posts, would something automatic help you with alt text checking (for instance noting when it's missing)?

**Comment by maelle** [comment]
actually there's a rolint for this already :woman_facepalming:
